api/v1beta1/flowcollector_types.go

```diff
@@ -329,9 +329,9 @@ type FLPMetrics struct {
 	Server MetricsServerConfig `json:"server,omitempty"`
 
 	// `ignoreTags` is a list of tags to specify which metrics to ignore. Each metric is associated with a list of tags. More details in https://github.com/netobserv/network-observability-operator/tree/main/controllers/flowlogspipeline/metrics_definitions .
-	// Available tags are: `egress`, `ingress`, `flows`, `bytes`, `packets`, `namespaces`, `nodes`, `workloads`, `nodes-flows`, `namespaces-flows`, `workloads-flows`.
+	// Available tags are: `egress`, `ingress`, `flows`, `bytes`, `packets`, `namespaces`, `nodes`, `workloads`, `resources`, `nodes-flows`, `namespaces-flows`, `workloads-flows`, `resources-flows`.
```

Comment:
```suggestion
	// Warning: removing tags from this list will result in more metrics creation, which can alter cluster performance and stability. It is recommended to monitor the memory and CPU usage of the `prometheus-k8s` deployment when doing so.
	// Available tags are: `egress`, `ingress`, `flows`, `bytes`, `packets`, `namespaces`, `nodes`, `workloads`, `resources`, `nodes-flows`, `namespaces-flows`, `workloads-flows`, `resources-flows`.
```

cc @skrthomas for doc review :-)

Reply:
Small suggestion to avoid "will" because peer reviewers will nab it: 

Warning: The removal of tags from this list results in increased metrics creation, which can...`prometheus-k8s` deployment when removing tags.

Reply:
> Small suggestion to avoid "will" because peer reviewers will nab it

I'll learn it eventually

Reply:
Its allll good :) I find my own PRs having it all the time. That's what peer reviews are for!